### PR TITLE
Home Page Created

### PR DESCRIPTION
**Pull request by aamrapaliwandhre**
**1)NavBar
2)HomePage
3)Footer**

Screenshot (81)
Screenshot (82)

### CODE REVIEW

File: Pages/HomePage/homepage.css [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+
+.overlay-container{
+    position: absolute;
+    left: 15%;
+    top: 35%;
+}
+
+.button-primary{
+    background-color:var(--black);
+    padding: 0.5rem 2rem 0.5rem 2rem;
```
Comment (pratik-kumar-sahu) [inline Pages/HomePage/homepage.css]: you can do like this also `padding: 0.5rem 2rem`

File: Pages/HomePage/homepage.css [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+}
+
+.button-primary{
+    background-color:var(--black);
+    padding: 0.5rem 2rem 0.5rem 2rem; 
+}
+
+.text-overlay{
+    margin: 70px auto;
+    background: #fff;
```
Comment (pratik-kumar-sahu) [inline Pages/HomePage/homepage.css]: move color to variable

File: Pages/HomePage/homepage.css [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+    border-radius: 5px;
+    text-align: center;
+    margin: 2rem 2rem 2rem 2rem;
```
Comment (pratik-kumar-sahu) [inline Pages/HomePage/homepage.css]: equivalent to `margin: 2rem`

**Review by pratik-kumar-sahu — APPROVED** [review]
- Else looks good go ahead with the merge

**Comment by ashifkhn** [comment]
Looks good. If you can adjust the placement for search it might look better.

File: Assets/cart.svg [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (aman11s) [inline Assets/cart.svg:1]: Instead of SVG images, you can use icons.

**Review by aman11s — APPROVED** [review]

File: Assets/cart.svg (hunk 4 above)
Comment (aamrapaliwandhre) [reply Assets/cart.svg:1]: i have used icon...forget to remove images from file

**Review by harshkotkar5557 — COMMENTED** [review]
Looks good to me

**Pull request merged**